subtests/docker_cli/selinux_labels/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
A bit more duplication than I care for; what would you think of http://pastebin.test.redhat.com/423099